examples/dreambooth/train_dreambooth_lora.py

```diff
@@ -70,6 +70,38 @@
 logger = get_logger(__name__)
 
 
+def text_encoder_lora_state_dict(text_encoder):
+    state_dict = {}
+
+    def text_encoder_attn_modules(text_encoder):
+        from transformers import CLIPTextModel, CLIPTextModelWithProjection
```

Comment:
Why is this needed? We can directly import it at the top.

Reply:
So that we can remove `text_encoder_attn_modules` from src/diffusers